charts/brig/templates/tests/brig-integration.yaml

```diff
@@ -37,6 +37,9 @@ spec:
     - name: "brig-secrets"
       secret:
         secretName: "brig"
+    - name: "turn-servers"
+      configMap:
+        name: "turn"
```

Comment:
I believe this can use the turn configmap from the chart itself? Or do I have to copy the configmap into the `tests` dir?